src/microbe_stage/BackgroundPlane.cs

```diff
@@ -0,0 +1,263 @@
+using System;
+using Godot;
+
+/// <summary>
+///   Manages the microbe background plane, optionally applies blur.
+/// </summary>
+public partial class BackgroundPlane : Node3D, IGodotEarlyNodeResolve
```

Comment:
Why does this use `IGodotEarlyNodeResolve` as this shouldn't be able to be loaded from saves due to being ignored in the camera class? As a result any saved state wouldn't work in this class whatsoever so this needs to rely on the camera to remember and re-set everything after load (as this is currently setup).